### PR TITLE
fix: serious grammar and spelling errors in docs 

### LINKED ISSUE #432: docs: serious grammar and spelling errors
# 🎯 Problem to be solved

The current documentation contains grammatical and spelling errors, as well as typos, which may hinder understanding and reduce the overall quality of the documentation. It is important to correct these errors to improve clarity and professionalism.

- [ ] Link to the codebase issue that prompted this docs change

# 🛠️ Proposed solution

- [ ] For major overhaul of docs, design doc link: 

# 🧪 Tests

- [ ] Netlify deployment build tests passing

# 👐 Additional acceptance criteria

# ❌ Out of Scope

### PR DESCRIPTION
**Pull request by antews**
## Summary

In reviewing the project documentation, I have identified and corrected several errors and typos that were inadvertently made during its creation. These errors can lead to confusion or misunderstanding. 

## Details

Ensuring clarity and accuracy in documentation is critical for both users and developers. 

Closes #432 

ticket:
#432

### CODE REVIEW

File: docs/advanced/quickstart-builder-api.mdx [hunk 1]
```diff
@@ -156,10 +157,10 @@ When your cluster is running, you should see if Charon is logging something like
...
+This indicates that your Charon node is successfully registering with the relay for a blind block when the time comes.
 
 If you are using the [ultrasound relay](https://relay.ultrasound.money), you can enter your cluster's distributed validator public key(s) into their website, to confirm they also see the validator as correctly registered.
 
-You should check that your validator client's logs look healthy, and ensure that you haven't added a `fee-recipient` address that conflicts with what has been selected by your cluster in your cluster-lock file, as that may prevent your validator from producing a signature for the block when the opportunity arises. You should also confirm the same for all of the other peers in your cluster.
+You should check that your validator client's logs look healthy, and ensure that you haven't added a `fee-recipient` address that conflicts with what has been selected by your cluster in your `cluster-lock.json file`, as that may prevent your validator from producing a signature for the block when the opportunity arises. You should also confirm the same for all of the other peers in your cluster.
```
Comment (OisinKyne) [inline docs/advanced/quickstart-builder-api.mdx:164]:
```suggestion
You should check that your validator client's logs look healthy, and ensure that you haven't added a `fee-recipient` address that conflicts with what has been selected by your cluster in your `cluster-lock.json` file, as that may prevent your validator from producing a signature for the block when the opportunity arises. You should also confirm the same for all of the other peers in your cluster.
```

File: docs/advanced/quickstart-builder-api.mdx [hunk 2]
```diff
@@ -156,10 +157,10 @@ When your cluster is running, you should see if Charon is logging something like
 13:10:47.094 INFO bcast      Successfully submitted validator registration to beacon node {"delay": "24913h10m12.094667699s", "pubkey": "84b_713", "duty": "1/builder_registration"}
 ```
 
-This indicates that your Charon node is successfully registering with the relay for a blinded block when the time comes.
+This indicates that your Charon node is successfully registering with the relay for a blind block when the time comes.
```
Comment (OisinKyne) [inline docs/advanced/quickstart-builder-api.mdx:160]:
```suggestion
This indicates that your Charon node is successfully registering with the relay for a blinded block when the time comes.
```
They are called blinded blocks. https://ethereum.github.io/beacon-APIs/#/Validator/produceBlindedBlock

**Review by OisinKyne — APPROVED** [review]

**Comment by OisinKyne** [comment]
Hey i can't merge this due to conflicts, can you figure out does it need a rebase or what is up with it? :)

**Review by OisinKyne — COMMENTED** [review]
Needs rebase

**Review by OisinKyne — CHANGES_REQUESTED** [review]
Needs rebase

**Comment by antews** [comment]
> Hey i can't merge this due to conflicts, can you figure out does it need a rebase or what is up with it? :)

Hi! @OisinKyne I've rebased and resolved all conflicts, now PR is ready for merge! Thanks!

**Comment by antews** [comment]
> Hi! @OisinKyne I've rebased and resolved all conflicts, now PR is ready for merge! Thanks!

Hi! @OisinKyne  I've rebased and resolved all conflicts, now PR is ready for merge! Thanks!

**Comment by OisinKyne** [comment]
Hey apologies. I was in a hurry to get these changes in before I tagged the next release such that they would show up outside the `next` section, so i implemented them in another PR. This PR just has two extra newlines left. Appreciate the follow up and rebase though, sorry for not coming back to this!

**Comment by antews** [comment]
> Hey apologies. I was in a hurry to get these changes in before I tagged the next release such that they would show up outside the `next` section, so i implemented them in another PR. This PR just has two extra newlines left. Appreciate the follow up and rebase though, sorry for not coming back to this!

Now I'am won't be in the contributor list, right? @OisinKyne

**Comment by antews** [comment]
> Hey apologies. I was in a hurry to get these changes in before I tagged the next release such that they would show up outside the `next` section, so i implemented them in another PR. This PR just has two extra newlines left. Appreciate the follow up and rebase though, sorry for not coming back to this!

?

**Comment by slugmann321** [comment]
@antews please only contribute if you actually want to see changes, not to get on a list. There is no "list" and being a contributor will not advantage you in any way. Thank you.

**Pull request closed without merging**